cmake/Modules/LibAddPlugin.cmake

```diff
@@ -351,8 +351,6 @@ function (add_plugin PLUGIN_SHORT_NAME)
 	add_library (${PLUGIN_OBJS} OBJECT ${ARG_SOURCES})
 	add_dependencies (${PLUGIN_OBJS} kdberrors_generated)
 
-	add_dependencies(${PLUGIN_OBJS} readme_${PLUGIN_SHORT_NAME}.c)
```

Comment:
> As far as I can tell, rebuilding works fine even after I delete the dependency.

It did not not even work before this PR, I reported the problem in #1560

Thus this PR does not change the behavior (it did not work before nor after), you do not have to worry about it.